src/components/cuda/tests/Makefile

```diff
@@ -22,6 +22,8 @@ endif
 
 PAPI_FLAG = -DPAPI    # Comment this line for tests to run without PAPI profiling
 NVCFLAGS += -g -ccbin='$(CC)' $(PAPI_FLAG)
+# Uncomment this line to fix CUDA path in executable, comment to rely on LD_LIBRARY_PATH
+# NVCFLAGS += -Xlinker "-rpath,$(PAPI_CUDA_ROOT)/lib64"
```

Comment:
What is this?

Reply:
I made this a comment to have a discussion regarding whether we want this or not.

Reply:
please remove. if you want to discuss we can do internally.

Reply:
Removed